tests/Backend/CommonPart1/DeleteRowsTest.php

```diff
@@ -431,7 +383,7 @@ public function testDeleteByValuesInWorkspaceWithInvalidData(): void
             $this->fail('Should fail because of invalid column type');
         } catch (ClientException $e) {
             $this->assertSame('Cannot use column "ID" to delete by. Column types do not match. Type is "NUMBER" but expected type is "VARCHAR".', $e->getMessage());
-            $this->assertSame('storage.tables.validation.invalidColumnToDeleteBy', $e->getStringCode());
+            $this->assertSame('storage.tables.invalidColumnToDeleteBy', $e->getStringCode());
```

Comment:
todle je sice BC break, ale je to nova exception a nemel by to snad byt problem. At je to narovnane s `storage.tables.columnNotExists`